source/Sashimi.Azure/Web/AzureApi.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Net.Http;
+using Octopus.Server.Extensibility.Extensions.Infrastructure.Web.Api;
+
+namespace Sashimi.Azure.Web
+{
+    public class AzureApi : RegisterEndpoint
+    {
+        public AzureApi(Func<SecuredAsyncActionInvoker<AzureEnvironmentsListAction>> azureEnvironmentsListInvokerFactory)
```

Comment:
@andyinaus I don't think this endpoint needs to be `SecuredAsyncActionInvoker`

Reply:
It's currently secured in the `Octopus.Server`. Do we wanna break it ?

Reply:
yes, it should not need to be secure